File: Logic Building Problems/Easy Problems/Program to calculate distance between two points/method2.java
// Java code to compute distance

class Main {
	// Function to calculate distance
	static double distance(int x1, int y1, int x2, int y2)
	{
		// Calculating distance
		return Math.sqrt(Math.pow(x2 - x1, 2)
						+ Math.pow(y2 - y1, 2) * 1.0);
	}
	// Driver code
	public static void main(String[] args)
	{
		System.out.println(
			Math.round(distance(3, 4, 4, 3) * 100000.0)
			/ 100000.0);
	}
}

File: Logic Building Problems/Easy Problems/Program to calculate distance between two points/method2.js

// Function to calculate distance 
function distance(x1, y1, x2, y2) 
{ 
	// Calculating distance 
	return Math.sqrt(Math.pow(x2 - x1, 2) + 
				Math.pow(y2 - y1, 2) * 1.0); 
} 

// Drivers Code 
document.write(distance(3, 4, 4, 3)); 
